tests/conftest.py

```diff
@@ -158,3 +159,47 @@ def mock_timepix_exceeded_buffer_run(mock_sqs_timepix_directory):
             size_dset[np.argmax(size_dset)] += tpx_root['data/x'].shape[1]
 
         yield RunDirectory(td).deselect('SQS_EXTRA*')
+
+@pytest.fixture(scope='session')
+def mock_sqs_grating_calibration_directory():
+    #energy = mock_grating_mono_energies()
+    energy = list()
+    for e in np.linspace(992.0, 1008.0, 10):
+        energy += [e]*20
+    energy = np.array(energy)
+
+    pix = np.linspace(0, 1000, 1000)
+
+    # convert energy to time of flight for etofs
+    # calibration constants
+    #offset, slope = mock_grating_calibration_constants()
+    offset, slope = 990, 20/1000.0
+    true_pix_to_e = lambda p: offset + p*slope
+    true_e_to_pix = lambda e: (e - offset)/slope
+    data = np.exp(-0.5*(pix[None, :] - true_e_to_pix(energy)[:,None])**2)
+    # make it 2D
+    data2d = np.stack([data]*100, axis=1)
+    # add dimension for pulses in Gotthard
+    data_gh = np.stack([np.zeros_like(data)]*20+[data]*100, axis=-2)
+
+    sources = [
+        Timeserver('SQS_RR_UTC/TSYS/TIMESERVER'),
+        MonoMdl('SA3_XTD10_MONO/MDL/PHOTON_ENERGY', energy_data=energy),
+        CameraWithData("SQS_DIAG3_BIU/CAM/CAM_6", data=data2d),
+        GotthardIIWithData("SQS_EXP_GH2-2/CORR/RECEIVER", data=data_gh),
```

Comment:
Can you estimate how much data this is? We should be a bit careful with (actual) mockdata to not grow out of proportion.

Reply:
I have reduced the number of pixels in one of the dimensions by a factor of 10 and with that the tests still pass, and the temporary run produced is only 22 MB. I doubt this is a large amount and surely the resources available would be able to cope with it.